cirq/google/api/v1/programs_test.py

```diff
@@ -20,7 +20,6 @@
 import cirq.google as cg
 import cirq.google.api.v1.programs as programs
```

Comment:
Do you want v2 programs here?

Reply:
Presumably that is tested here:
https://github.com/quantumlib/Cirq/blob/master/cirq/google/api/v2/program.py